community/cypher/cypher/src/test/scala/org/neo4j/cypher/SemanticDeleteAcceptanceTest.scala

```diff
@@ -40,15 +40,16 @@ class SemanticDeleteAcceptanceTest extends ExecutionEngineFunSuite with PatternG
 
       whenever(pattern.nonEmpty) {
         val patternString = pattern.map(_.string).mkString
-
-        //update
-        updateWithBothPlanners(s"CREATE $patternString")
-        //delete
-        val identifiers = findAllRelationshipNames(pattern) ++ findAllNodeNames(pattern)
-        updateWithBothPlanners(s"MATCH $patternString DELETE ${identifiers.mkString(",")} RETURN count(*)")
-
-        //now db should be empty
-        executeWithAllPlanners("MATCH () RETURN count(*) AS c").toList should equal(List(Map("c" -> 0)))
+        withClue(s"failing on pattern $patternString") {
+          //update
+          updateWithBothPlanners(s"CREATE $patternString")
+          //delete
+          val identifiers = findAllRelationshipNames(pattern) ++ findAllNodeNames(pattern)
+          updateWithBothPlanners(s"MATCH $patternString DELETE ${identifiers.mkString(",")} RETURN count(*)")
```

Comment:
No we should not do that, doing exactly that has caused us a ton of errors with eagerness.  `updateWithBothPlanners` asserts that both rule planner and cost planner returns the same thing.